web/client/src/lib/countryCodes.js

```diff
@@ -0,0 +1,31 @@
+const language = navigator.language || 'en-US'
+
+function getCountries() {
```

Comment:
The current default is pulled via what the locale of the browser is configured as. Or at least it is supposed to. 

I misunderstood that the desired behavior was for this filed to be optional. 

I'll attempt to refactor to use a default blank value, and also populate the auto-detected based on browser country code as the second option.

Reply:
Thanks @KClough! Yes, country code is optional in the desktop app, so we want it to be optional here. Your idea to default to browser country code would be better than the blank value.

Reply:
@emmambd I just committed a fix for the first issue in this thread, the `NoSuchKey` error should no longer occur on subsequent runs.

Reply:
I've made country code optional, the dropdown now defaults to "Choose a region" and places the detected region, if any, as the first option in the list. 

The web service also no longer sets a default region if one is not provided.

Reply:
I've pushed some tweaks to the cloud run instance configuration (4vcpu + 16gb ram) and updated the JVM to use 12gb ram. 

This may require additional tweaking as we expose this to real traffic, but as of now this configuration allows the file linked above (#3) [to process](https://gtfs-validator-results.mobilitydata.org/a0f43999-857b-4792-be8f-702f0ee4337b/report.html). 

We also noticed that there were some out of memory errors thrown with the latest release of the desktop app. Issue logged [here](https://github.com/MobilityData/gtfs-validator/issues/1358).

Here are my testing results with the CLI, Desktop and web versions:

**Local CLI:**
-Xmx6g :x:
-Xmx7g :white_check_mark:
-Xmx8g :white_check_mark:

**Web validator configured with 8gb of memory:**
-Xmx7g  :x:
-Xmx8g  :x:

**Web validator configured with 16gb of memory:**
-Xmx8g  :white_check_mark:

Reply:
re: the optional region and URL validation report bug, thank you @KClough and @briandonahue for these fixes! I've re-tested and both are resolved. The optional region also works for different browser language presets.